src/Tickets/Commerce/Gateways/Stripe/Payment_Intent_Handler.php

```diff
@@ -215,4 +210,46 @@ public function delete_payment_intent_transient() {
 	public function store_payment_intent( $payment_intent ) {
 		set_transient( $this->get_payment_intent_transient_name(), $payment_intent, 6 * HOUR_IN_SECONDS );
 	}
+
+	/**
+	 * Get the additional metadata for the payment intent.
+	 *
+	 * @since TBD
+	 *
+	 * @param \WP_Post $order The Order data.
+	 * @param array $payment_intent The Payment intent.
+	 *
+	 * @return array
```

Comment:
Maybe we can document a bit the format of the array, or the keys that it should contain?

Reply:
We don't have any required set of metadata, anything with a key-value pair should be valid as long as the hooked filter is appending data to these values.